octopoes/xtdb/client.py

```diff
@@ -103,6 +104,24 @@ def submit_transaction(self, operations: List[Operation]) -> None:
         self._verify_response(res)
         self.await_transaction(res.json()["txId"])
 
+    def create_node(self, name: str) -> None:
+        res = self._session.post(
+            "/create-node",
+            data=json.dumps({"node": name}),
+            headers={"Content-Type": "application/json"},
+        )
+
+        self._verify_response(res)
+
+    def delete_node(self, name: str) -> None:
+        res = self._session.post(
+            "/delete-node",
+            data=json.dumps({"node": name}),
```

Comment:
Maybe directly use the json attribute? This will inject the correct content-type headers automatically, and removed the need to dump to json first.